developer/src/kmc-ldml/src/compiler/messages.ts

```diff
@@ -7,16 +7,16 @@ const SevError = CompilerErrorSeverity.Error | CompilerErrorNamespace.LdmlKeyboa
 const SevFatal = CompilerErrorSeverity.Fatal | CompilerErrorNamespace.LdmlKeyboardCompiler;
 
 export class CompilerMessages {
-  static Error_InvalidNormalization = (o:{form: string}) => m(this.ERROR_InvalidNormalization, `Invalid normalization form '${o.form}`);
-  static ERROR_InvalidNormalization = SevError | 0x0001;
+  static Hint_NormalizationDisabled = () => m(this.HINT_NormalizationDisabled, `normalization=disabled is not recommended.`);
+  static HINT_NormalizationDisabled = SevHint | 0x0001;
```

Comment:
Again, this can still be troublesome if one cannot disable because it means you can never get to a clean build. It's hard to know what to do with this -- barring adding message suppression to the compiler toolchain

Reply:
do you want to back it out?

Reply:
Perhaps, and we should then open an issue on linting and use this as a first datapoint.